src/platform/PlatformTools.ts

```diff
@@ -161,8 +155,16 @@ export class PlatformTools {
         return fs.readFileSync(filename);
     }
 
-    static appendFileSync(filename: string, data: any): void {
-        fs.appendFileSync(filename, data);
+    /**
+     * Asynchronously append data to a file, creating the file if it does not exist.
+     *
+     * @param path A path to a file.
+     * @param data The data to write. If something other than a Buffer or Uint8Array is provided, the value is coerced to a string.
+     */
+    static appendFile(path: string, data: any) {
+        return new Promise<void>((resolve, reject) => {
```

Comment:
node.js has builtin method `promisify`, I would suggest either use it or `fs-extra` instead

Reply:
Yes, node 8+. Why add a dependency just to return a promise? Sounds a bit overkill

Reply:
I meant that promisify is a node util, well tested and working solution, why do you need to implement this functionality by yourself even with a simple promise? As for fs-extra - it's a replacement of fs module, it makes sense to use it only if you have many places like this

Reply:
We have many places like this. Many places. And since we can't use external modules without calling PlatformTools it would be better to create a local function instead. But Im quite satisfied with current approach.